services/wallet/router/router_v2.go

```diff
@@ -305,6 +305,12 @@ func validateInputData(input *RouteInputParams) error {
 		return nil
 	}
 
+	if input.SendType == ENSRelease {
```

Comment:
is there anyway to check that in the processor via the can?
That feels very processors focused?

Reply:
Isn't late then? I just wanted to validate input data at the very first stage of calling the router?